hubverse_annotator/utils.py

```diff
@@ -14,15 +14,20 @@
 from typing import Literal
 
 import altair as alt
+import colorbrewer
```

Comment:
Rather than colorbrewer, we should try to take advantage of https://vega.github.io/vega/docs/schemes/#blues